Cython/Includes/libc/limits.pxd

```diff
@@ -26,3 +26,8 @@ cdef extern from "<limits.h>":
     const long long LLONG_MIN
     const long long LLONG_MAX
     const unsigned long long ULLONG_MAX
+
+    # This isn't a type defined by C but by Python. Still, this seems the
+    # most sensible place to put these constants
```

Comment:
Hmm … there are also other defines like `PY_INT64_T`, which also don't belong here. Add a `cpython.pyport`, maybe?

Reply:
> Add a `cpython.pyport`, maybe?

I was considering `cpython.types`

Reply:
They are defined in `pyport.h`, and `types` seems too similar to `type`.